detect_lang/detect_lang.py

```diff
@@ -0,0 +1,27 @@
+from langdetect import DetectorFactory, detect, detect_langs
+import time
+
+def get_message_lang(message_text, user_target_lang, user_native_lang):
+    try:
+        # return target lang if message is target lang
+        if detect(message_text) == user_target_lang:
+            return user_target_lang
+
+        # return native lang if message is native lang
+        elif detect(message_text) == user_native_lang:
+            return user_native_lang
+
+        # if native/target lang is not detected (due to langdetect probability algorithm) then generate new probabilities
+        # until native/target lang is detected
+        else:
+            init_time = time.time()
+            while True:
+                # if native/target lang is not detected after five seconds, return None
+                if (time.time()-init_time) >= 5:
```

Comment:
Thank you for this. Just curious, why did you decide to go with the approach of limiting this with time? (As opposed to the number of loops, for example)